experiments/scripts/subj-scripts/crystal.sh

```diff
@@ -0,0 +1,19 @@
+# The root directory for the old subject
+export DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/crystalvc
+# Directory for where all of the old subject's information is stored.
+export DT_SUBJ=$DT_SUBJ_ROOT
+# Directory containing the old subject's src directory.
+export DT_SUBJ_SRC=$DT_SUBJ_ROOT
+
+# The root directory for the new subject
+export NEW_DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/crystal
+# Directory for where all of the new subject's information is stored.
+export NEW_DT_SUBJ=$NEW_DT_SUBJ_ROOT
+# Directory containing the new subject's src directory.
+export NEW_DT_SUBJ_SRC=$NEW_DT_SUBJ_ROOT
+
+# The name of the subject you wish to add (e.g., xml_security)
+export SUBJ_NAME=crystal
+# The name of the subject you want to be displayed in the paper (e.g., XML Security)
+export SUBJ_NAME_FORMAL=Crystal
+
```

Comment:
Please have this script and the other ant ones `source ../setup-vars-ant.sh` so that the user won't have to do one more step to setup the variables.